static/scripts/app-register.js

```diff
@@ -73,13 +73,10 @@ formbtn.addEventListener('click', async (e) => {
     
         let result = await datafetch.json();
 
-            
-
-            
-
-        if(result.status_code === 200) {
-                console.log('OK')
-        } else if(result.status_code === 409){
+        if(datafetch.ok) {
+            console.log(datafetch.status)
```

Comment:
Why not make it a switch statement on response status?

Reply:
coz i use switch for short tasks like:
switch(something){
case 1:
return test (1-2 lines)
case 2:
return test2
...

```
but in our example u return :
let user_err = document.createElement('small');
                    user_err.classList.add('user-err');
                    user_err.innerText = `Username ${userNameInput.value} is already exist`;
                    userNameInput.style.border = '2px solid #b90909'
                    // remove err if already exist
                    if(userNameInput.nextElementSibling.classList.contains('user-err')){
                        userNameInput.nextElementSibling.remove();
                    }

                    userNameInput.after(user_err);
                    setTimeout(() => {
                        user_err.remove();
                        userNameInput.style.border = '2px solid #dddddd';
                        userNameInput.addEventListener('focus', () => userNameInput.style.border = '2px solid #0941b9');
                        userNameInput.addEventListener('blur', () => userNameInput.style.border = '2px solid #dddddd')
                    }, 5000);
            console.log(datafetch.statusText, 'Username already exist!')
```

            which is not fit!